docs/DEVELOPING.md

```diff
@@ -138,12 +157,54 @@ The **body** provides more details, it should include the motivation for the cha
 
 Just as in the subject, use the imperative, present tense: "change" not "changed" nor "changes"a
 
+Paragraphs or bullet points are ok (must not exceed 100 characters per line). Typically a hyphen or asterisk is used for the bullet, followed by a single space, with blank lines in between, but conventions vary here.
+
 #### footer (optional)
 
 The **footer** should contain any information about Breaking Changes introduced by this commit.
 
 This section must start with the upper case text `BREAKING CHANGE` followed by a colon (`:`) and a space (` `). A description must be provided, describing what has changed and how to migrate from older versions.
 
+### Tools to help generate a commit message
+
+This repository has [commitizen](https://github.com/commitizen/cz-cli) support enabled. Commitizen can help you generate your commit messages automatically. You must install it globally as follows:
+
+```sh
+$ npm i -g commitizen
+```
+
+And to use it, simply call `git cz` instead of `git commit`. The tool will help you generate a commit message that follows the above guidelines.
```

Comment:
Were *the above guidelines* directly from `commitizen` or is there some kind of config that sets things like the 100 char "limit".

Might be good to mention where "the above guidelines" come from.

Reply:
Most of the guidelines are from https://conventionalcommits.org/, some tweaks are our own (100 char limit, list of scopes defined by monorepo packages). We do mention https://conventionalcommits.org/ at the beginning of _Commit message guidelines_ section. I feel that's good enough.